go.mod

```diff
@@ -35,9 +35,11 @@ require (
 	github.com/pierrec/lz4 v2.0.5+incompatible // indirect
 	github.com/ryanuber/go-glob v1.0.0 // indirect
 	github.com/sirupsen/logrus v1.4.1 // indirect
+	github.com/spiffe/go-spiffe v0.0.0-20190518123159-37d000f27824
 	golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3 // indirect
 	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
+	google.golang.org/appengine v1.4.0 // indirect
```

Comment:
Looks like this snuck in somehow as well. Frankly, I don't know _how_ `go mod tidy` works this out, it doesn't seem to me to be deterministic, but I think we should be able to remove this.

Reply:
Yeah that was weird. Did some testing and looks like go fmt adds that for some reason? Running fmt then tidy means no appengine in go.mod, running tidy then fmt means appengine in go.mod. I'm very confused.

Fixed though.